go/libraries/doltcore/dtestutils/testcommands/command.go

```diff
@@ -48,7 +48,10 @@ func (a StageAll) Exec(t *testing.T, dEnv *env.DoltEnv) error {
 	roots, err := dEnv.Roots(context.Background())
```

Comment:
we need to kill off this package, I added `testcommands` before realizing we could just use the actual commands in tests

Reply:
added TODO